tools/axe-protractor/axe-protractor.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+/*
+ * Protractor Plugin to run axe-core accessibility audits after Angular bootstrapped.
+ */
+
+const AxeBuilder = require('axe-webdriverjs');
+const {buildMessage} = require('./build-message');
+
+/* Resolved violations */
+const violations = {};
+
+/** When the page is ready and Angular is bootstrapped. */
+function onPageStable() {
+  AxeBuilder(browser.driver)
+    .configure(this.config || {})
+    .analyze(results => handleResults(this, results));
+}
+
+/** Handles the results of axe-core. */
+function handleResults(context, results) {
```

Comment:
`handleAxeResults`?
Also expand the comments with some explanation of what we're doing with those results.